draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2194,7 +2207,7 @@ uCDN-A                         dCDN-B                         dCDN-C
         <vspace blankLines="1"/>
         Value: TimeWindow object using UNIX epoch timestamps (see Section 4.2.3.2 of <xref target="RFC8006"/>).
         <vspace blankLines="1"/>
-        Mandatory: No, but exactly one of "unixEpochWindow" or "utcWindow" MUST be present.
+        Mandatory: No, but exactly one of the "unixEpochWindow" or "utcWindow" MUST be present.
```

Comment:
Adding "the" sounds more awkward here.  We're referring to one of these choices.  Maybe "either" is is a better choice than "the".